schema.graphql

```diff
@@ -189,3 +226,40 @@ type ZTokenStatus @entity {
   blockNumber: BigInt!
   timestamp: BigInt!
 }
+
+type Setting @entity {
+  id: ID!
+  settingName: SettingName!
+  value: BigInt!
+  lastBlockNumber: BigInt!
+  lastTimestamp: BigInt!
+}
+
+type SettingsStatus @entity {
+  id: ID!
+  transaction: EthTransaction!
+  oldValue: BigInt!
+  newValue: BigInt!
+  from: Bytes!
+  settingName: SettingName!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+}
+
+type LendingPoolPause @entity {
```

Comment:
Isn't the lendingPool entity has a status itself, why you would require a secondary model to store if a LendingPool is paused? (or this is to store the action and you can relate these pause events with the lendingPools?)